docs/community.md

```diff
@@ -1,6 +1,6 @@
 # Community
 
-Welcome to the Secure Device Onboard community. Secure Device Onboard is an open source project sponsored by [LF Edge](https://www.lfedge.org/projects/securedeviceonboard/).
+Welcome to the FIDO Device Onboard community. FIDO Device Onboard is an open source project sponsored by [LF Edge](https://www.lfedge.org/projects/securedeviceonboard/).
 
 Community practices are documented at the [Secure Device Onboard wiki](https://wiki.lfedge.org/display/SDO/Secure+Device+Onboard).
```

Comment:
Following community links still holds SDO, those can also be changed.

Reply:
> Following community links still holds SDO, those can also be changed.

the project link is now /projects/fidodeviceonboard